.azure-pipelines/ipu-tests.yml

```diff
@@ -76,7 +76,8 @@ jobs:
         source ${{ variables.poplar_sdk }}/poplar-ubuntu*/enable.sh
         source ${{ variables.poplar_sdk }}/popart-ubuntu*/enable.sh
         export POPTORCH_WAIT_FOR_IPU=1
-        python -m coverage run --source pytorch_lightning -m pytest tests/accelerators/test_ipu.py -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
```

Comment:
Note that there are no ipu tests outside of this file. I personally prefer the simplicity of calling a single file. It's also faster as you don't need to run the test collection on the entire codebase.

This could be added if there's a desire to add more ipu tests around the test suite

Reply:
Just realised that the following test was missed because we manually add the test file here.
```
tests/deprecated_api/test_remove_1-8.py::test_v1_8_0_deprecated_training_type_plugin_classes[IPUPlugin]
```

It can easily happen again as we add more tests across multiple files in the future. IMHO, stability/reliability is more important than optimization here.